odoo/addons/base/data/res_currency_data.xml

```diff
@@ -1545,6 +1545,11 @@
             <field name="currency_unit_label">Quetzal</field>
             <field name="currency_subunit_label">Centavo</field>
         </record>
+	
+	<record id="rate_precision" model="decimal.precision">
```

Comment:
@alxolivares The runbot is red. I think you should to add the forcecreate=True in the decimal precision

``` xml
<record forcecreate=True id="rate_precision" model="decimal.precision">
```